### ISSUE TITLE
Onboarding COMMUNITY WHY video / Interview community members

### ISSUE THREAD

**Issue by tamarandom** [CommonsBuild/coordination#309]
Interview List https://docs.google.com/document/d/1RSzWsn24IFFlik6yjmEAtZb4-HM9Cgf_WE8cJljOGKU/edit

**Comment by markoprljic** [comment]
@eduardovegap This one?

**Comment by jessicazartler** [comment]
I am storyboarding and finalizing questions to begin setting up the interviews next week, will ask for help from Eduardo when I need assistance

**Comment by markoprljic** [comment]
Hi @jessicazartler how are we progressing with the interviews?

**Comment by jessicazartler** [comment]
I am posting around and contacting people to schedule some interviews this WED/FRI :D  Do you want to be interviewed? :D Thanks for checking in

**Comment by markoprljic** [comment]
@jessicazartler Yes, I'll was thinking to send you audio recording :)

**Comment by jessicazartler** [comment]
VIDEO! We wanna see yo face :D

**Comment by jessicazartler** [comment]
@Zeptimus @eduardovegap you mentioned you were up for supporting -- created this list of people/intvw questions - are you all up for taking some of these interviews and doing recordings then passing the video files for me to edit? 

Shouldn't take more than 15min call each to finish - DM me with questions or we can have a quick sync on this tomorrow :D
https://docs.google.com/document/d/1RSzWsn24IFFlik6yjmEAtZb4-HM9Cgf_WE8cJljOGKU/edit?usp=sharing

**Comment by Zeptimus** [comment]
Yeah, how do we coordinate it? im up to take some interviews even help with the editing part if needed

**Comment by markoprljic** [comment]
@jessicazartler Just checking in to see if you managed to coordinate with @Zeptimus on assisting with interviews and what's the progress status? Thanks.

**Comment by Zeptimus** [comment]
Hey marko all the progress its being added on this doc https://docs.google.com/document/d/1RSzWsn24IFFlik6yjmEAtZb4-HM9Cgf_WE8cJljOGKU/edit

**Comment by markoprljic** [comment]
Do you guys think we can set up calls for the remaining interviewees this week? @Zeptimus @eduardovegap @jessicazartler

**Comment by jessicazartler** [comment]
Yes, goal is to get them done by end of the week.. how you going Zepti & Eduardo?

**Comment by Zeptimus** [comment]
I have all exept Jess and Livi defenitly can do it by this week. I think i could do also Sem since i can sneak on his agenda

**Comment by markoprljic** [comment]
You can do with me @Zeptimus unless someone else already got assigned 😊

**Comment by Zeptimus** [comment]
Edu got you assigned but if there is any bandwith problems contact me i would love to do it :D

**Comment by jessicazartler** [comment]
My bandwidth is very low this week just to let you know, I have taken on 3 intvws and that's prob all I can do this week - thanks guys for supporting! Also maybe we can check with stewards about anyone else they feel is important to be in the video -- 15-20 is MORE than enough as we want to keep the video not too long and also - someone has to edit all these! haha thanks guys

**Comment by markoprljic** [comment]
I went ahead and interviewed myself :D Sending the recording to @jessicazartler Anyone else I should send it?

**Comment by jessicazartler** [comment]
Nope thanks Marko! Got it in my folder - very nice!!!! Thank you :D

**Comment by markoprljic** [comment]
Update from comms call:

- All materials will be completed this week
- @mount-manu will take it from there, cut all clips and suggest the creative direction (storyboard)
- no blockers to complete it this sprint

**Comment by markoprljic** [comment]
**Current status**

TO INTERVIEW:
Livia Deschermayer, TEC Steward (zepti)
Jessica Zartler, TEC Steward (zepti)
Sem Brestels (paulo)
Craig (edu)
Nate (edu)
Jeff Emmett, Token Engineering Researcher (jess) scheduled
Michael Zargham, Token Engineer (jess) scheduling for weekend

**Comment by Zeptimus** [comment]
Update:livi already done, Jess its going to self interview, and sem its super busy with the hatch (he is not at home)

**Comment by markoprljic** [comment]
@Zeptimus @jessicazartler @PJColombo @eduardovegap I'm seeing just a couple of interviews to be done. Wanted to let you know that Manu is being blocked by this to complete his issue on video editing. Would be great if we can get these done and passed over to him early next week. What do you think?

**Comment by mount-manu** [comment]
To clarify, I have the interviews from:

1. Angela
2. Eduardo
3. Griff
4. Jake
5. Lívia
6. Tamara
7. YGG
8. Marko
9. Santi
10. Sebnem
11. Simon
and 12. Zeptimus

Who's missing?

**Comment by Zeptimus** [comment]
> To clarify, I have the interviews from:
> 
> 1. Angela
> 2. Eduardo
> 3. Griff
> 4. Jake
> 5. Lívia
> 6. Tamara
> 7. YGG
> 8. Marko
> 9. Santi
> 10. Sebnem
> 11. Simon
> 12. Zeptimus
> 
> Who's missing?


I believe I also send durgadas ivy and Juan if you don't have them poke me and I re-send them to you, if u ask me Jess sem and more TE must be there 👯

**Comment by mount-manu** [comment]
> > To clarify, I have the interviews from:
> > 
> > 1. Angela
> > 2. Eduardo
> > 3. Griff
> > 4. Jake
> > 5. Lívia
> > 6. Tamara
> > 7. YGG
> > 8. Marko
> > 9. Santi
> > 10. Sebnem
> > 11. Simon
> > 12. Zeptimus
> > 
> > Who's missing?
> 
> I believe I also send durgadas ivy and Juan if you don't have them poke me and I re-send them to you, if u ask me Jess sem and more TE must be there 👯

Correct, my bad, I've updated the bespoke list: I've 15 interviews.

**Comment by jessicazartler** [comment]
Zargham is crazy busy trying to get him ASAP - Will record Jeff/I's interviews on Thursday 🙏

**Comment by jessicazartler** [comment]
FINALLY got Z yesterday! Will upload the video to our raw interviews folder, interviewing Jeff/me today so that is it for intvws from me!

**Comment by tamarandom** [comment]
@jessicazartler Close this then??

**Comment by jessicazartler** [comment]
Yes, thanks! Uploaded Z - doing the last two before Monday so consider it closed! Thank you
